i18n/en_AU.json

```diff
@@ -2105,7 +2105,7 @@
   "admin.sql.traceTitle": "SQL Statement Logging: ",
   "admin.support.aboutDesc": "The URL for the About link on the Mattermost login and sign-up pages. If this field is empty, the About link is hidden from users.",
   "admin.support.aboutTitle": "About Link:",
-  "admin.support.emailHelp": "Email address displayed on email notifications and during tutorial for end users to ask support questions.",
+  "admin.support.emailHelp": "Email address displayed on email notifications.",
```

Comment:
Please revert change to this file. We're only suppose to change from `en.json`. This (en_AU) and other locales will be handled separately by the Localization/Translation team.

Reply:
I have reverted it